pkgs/native_toolchain_c/lib/src/cbuilder/ctool.dart

```diff
@@ -46,6 +46,14 @@ abstract class CTool {
   /// The sources will be reported as dependencies of the hook.
   final List<String> includes;
 
+  /// Files passed to the compiler that will be included before all source
+  /// files.
+  ///
+  /// Resolved against [LinkInput.packageRoot].
+  ///
+  /// The sources will be reported as dependencies of the hook.
+  final List<String> forcedIncludes;
```

Comment:
Maybe add a link to a microsoft and clang documentation page.

Reply:
Added links in the documentation for the focedIncludes prop.